traffic-advice.bs

```diff
@@ -0,0 +1,265 @@
+<pre class="metadata">
+Title: Traffic Advice
+Shortname: traffic-advice
+Status: DREAM
+Repository: buettner/private-prefetch-proxy
+Editor: Jeremy Roman, Google https://www.google.com/, jbroman@chromium.org
+Abstract: A proposal to allow site owners to advice prefetch proxies and other agents to disallow traffic.
+Markup Shorthands: css no, markdown yes
+Assume Explicit For: yes
+Complain About: accidental-2119 yes, missing-example-ids yes
+Indent: 2
+Boilerplate: omit conformance
+</pre>
+<pre class="biblio">
+{
+    "HTTP-SEMANTICS": {
+        "aliasOf": "RFC7231"
+    },
+    "WELL-KNOWN": {
+        "aliasOf": "RFC8615"
+    }
+}
+</pre>
+
+Introduction {#intro}
+=====================
+
+*This section is non-normative.*
```

Comment:
If you wrap this section in `<div nonnormative>` you will get complaints about any accidental RFC 2119 keyword usage.

Reply:
The right syntax appears to be `class="non-normative"`. Done, and reworded a bunch of stuff to avoid adding the "this isn't really an RFC 2119 thing" spans everywhere.